flink-ml-servable-lib/src/test/java/org/apache/flink/ml/classification/LogisticRegressionModelServableTest.java

```diff
@@ -0,0 +1,142 @@
+                    Arrays.asList(
+                            new Row(
+                                    new ArrayList<>(
+                                            Arrays.asList(Vectors.dense(1, 2, 3, 4), 0., 1.))),
+                            new Row(
+                                    new ArrayList<>(
+                                            Arrays.asList(Vectors.dense(1, 2, 3, 4), 0., 2.))),
+                            new Row(
+                                    new ArrayList<>(
+                                            Arrays.asList(Vectors.dense(2, 2, 3, 4), 0., 3.))),
+                            new Row(
+                                    new ArrayList<>(
+                                            Arrays.asList(Vectors.dense(3, 2, 3, 4), 0., 4.))),
+                            new Row(
+                                    new ArrayList<>(
+                                            Arrays.asList(Vectors.dense(4, 2, 3, 4), 0., 5.))),
+                            new Row(
+                                    new ArrayList<>(
+                                            Arrays.asList(Vectors.dense(11, 2, 3, 4), 1., 1.))),
+                            new Row(
+                                    new ArrayList<>(
+                                            Arrays.asList(Vectors.dense(12, 2, 3, 4), 1., 2.))),
+                            new Row(
+                                    new ArrayList<>(
+                                            Arrays.asList(Vectors.dense(13, 2, 3, 4), 1., 3.))),
+                            new Row(
+                                    new ArrayList<>(
+                                            Arrays.asList(Vectors.dense(14, 2, 3, 4), 1., 4.))),
+                            new Row(
+                                    new ArrayList<>(
+                                            Arrays.asList(Vectors.dense(15, 2, 3, 4), 1., 5.)))));
+
+    private static final DenseVector COEFFICIENT = Vectors.dense(0.525, -0.283, -0.425, -0.567);
+
+    private static final double TOLERANCE = 1e-7;
+
+    @Test
+    public void testParam() {
+        LogisticRegressionModelServable servable = new LogisticRegressionModelServable();
+        assertEquals("features", servable.getFeaturesCol());
+        assertEquals("prediction", servable.getPredictionCol());
+        assertEquals("rawPrediction", servable.getRawPredictionCol());
+
+        servable.setFeaturesCol("test_features")
+                .setPredictionCol("test_predictionCol")
+                .setRawPredictionCol("test_rawPredictionCol");
+        assertEquals("test_features", servable.getFeaturesCol());
+        assertEquals("test_predictionCol", servable.getPredictionCol());
+        assertEquals("test_rawPredictionCol", servable.getRawPredictionCol());
+    }
+
+    @Test
+    public void testTransform() throws IOException {
+        LogisticRegressionModelData modelData = new LogisticRegressionModelData(COEFFICIENT);
```

Comment:
I am not sure we need this test given that we don't expect users to construct model data directly like this.

Note that we do not have a similar test for LogisticRegressionModel where we explicitly construct model data and give it to the LogisticRegressionModel.

Reply:
I think it is expected to construct model data like this, e.g. for users who want to test if the servable transformation logic is correct without training a model. 

We do not have a similar test for LogisticRegressionModel because the `LogisticRegression` and `LogisticRegressionModel` are in the same module, it is reasonable to validate `fit` and `transform` in the same test case. However, `LogisticRegressionModelServable` is in a separate module, which means it can be used and tested independently.

Reply:
The separate of `flink-ml-lib` and `flink-ml-servable-lib` is an implementation detail and should not affect how users use `Servable` classes, right?

Reply:
I agree. In this way, I could remove all tests in `flink-ml-servable-lib` currently. 

I've updated the PR based on your advice, could you have a look again?